public/components/ctas/8.html

```diff
@@ -0,0 +1,9 @@
+<div class="p-20 bg-sky-50 flex items-center flex-wrap justify-between gap-y-8">
```

Comment:
Need to consider mobile, `p-20` won't work.

Haven't seen the preview yet, but imagine `flex items-center flex-wrap justify-between gap-y-8` could be shrunk down.

Reply:
i get that done:)